lib/biran/configurinator.rb

```diff
@@ -53,7 +53,7 @@ def build_app_config
       }
 
       app_config.deep_merge! app_config_defaults
-      app_config[:secrets].deep_merge! get_secret_contents(app_config)
+      app_config[:secrets] = get_secrets_contents(app_config)
```

Comment:
Apparently, I forgot to test the scenario where there was nothing set in `app_config[:secrets]`. I think I was focused on priority and order and not fall back.